tests/mocks/database/population/__tests__/population.service.test.ts

```diff
@@ -268,10 +268,15 @@ describe('Population Service', function () {
         .forEach(({
           day, startTime, endTime, nonClassEvent,
         }) => {
-          const { contact, title } = nonClassEvent.nonClassParent;
+          const {
+            contactName, contactPhone, contactEmail, title, expectedSize,
+          } = nonClassEvent.nonClassParent;
           const testMeeting = testData.nonClassMeetings.find((nonClass) => (
             nonClass.title === title
-            && nonClass.contact === contact
+            && nonClass.contactName === contactName
+            && nonClass.contactPhone === contactPhone
+            && nonClass.contactEmail === contactEmail
+            && nonClass.expectedSize === expectedSize
```

Comment:
Should we include the `notes` here too?

Reply:
D'oh. Yes, we should. Fixed in 320a255f44fbc2d91ba58c54439a1efc3c81b062